.github/workflows/Documentation.yml

```diff
@@ -0,0 +1,17 @@
+name: Documentation
+
+on: [ push, pull_request ]
+
+jobs:
+  tests:
+    name: Render documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Test if the documentation will render without warnings
+        run: |
+          mkdir -p Documentation-GENERATED-temp \
+          && docker run --rm --pull always -v $(pwd):/project \
+             ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log
```

Comment:
Shouldn't this be `make test-docs` instead? Otherwise this is a pointless duplication of said `Makefile` target which increases the maintenance burden.

Reply:
I'm not sure about this one. Maybe @linawolf can help?